pkg/cmd/templates/templates.go

```diff
@@ -49,8 +51,11 @@ Examples:
 {{end}}{{end}}{{ if or $visibleFlags.HasFlags $explicitlyExposedFlags.HasFlags}}
 Options:
 {{ if $visibleFlags.HasFlags}}{{flagsUsages $visibleFlags}}{{end}}{{ if $explicitlyExposedFlags.HasFlags}}{{flagsUsages $explicitlyExposedFlags}}{{end}}{{end}}{{ if .HasSubCommands }}
-Use "{{$rootCmd}} <command> --help" for more information about a given command.{{end}}{{ if $optionsCmdFor}}
-Use "{{$optionsCmdFor}}" for a list of global command-line options (applies to all commands).{{end}}`
+Use "{{$rootCmd}} <command> --help" for more information about a given command.{{end}}
+{{ if $optionsCmdFor}}
```

Comment:
I believe this might be adding an unwanted line break above the new suggestion. Make sure we have just the wanted line breaks in all scenarios.